hopsworks-api/src/main/java/io/hops/hopsworks/api/jwt/JWTResource.java

```diff
@@ -64,10 +77,55 @@ public Response createToken(JWTRequestDTO jWTRequestDTO) throws NoSuchAlgorithmE
   @Path("/renew")
   public Response renewToken(JsonWebTokenDTO jsonWebTokenDTO) throws SigningKeyNotFoundException, NotRenewableException,
       InvalidationException {
-    JWTResponseDTO jWTResponseDTO = jWTHelper.renewToken(jsonWebTokenDTO);
+    JWTResponseDTO jWTResponseDTO = jWTHelper.renewToken(jsonWebTokenDTO, true, new HashMap<>(3));
     return Response.ok().entity(jWTResponseDTO).build();
   }
 
+  @PUT
+  @Path("/renew/service")
```

Comment:
We have a design document that we should stick to when writing endpoints. We should not put verbs in the endpoint name. That's not RESTful.

I'd rather have 
```
PUT -> /service -> renew the service token
DELETE -> /service/{token} -> Invalidate a token
```
Also it would be nice if add the swagger annotations.